app/models/customer.rb

```diff
@@ -0,0 +1,7 @@
+class Customer < ApplicationRecord
+
+  has_many :reservations, dependent: :destroy
+
+  validates :name, :email, presence: true
+  validates :email, uniqueness: true
```

Comment:
case_sensitive?

Reply:
Sir, it depends on database, we are using that it is case sensitive or not.
MySQL is bydefault case insensitive, so i don't have to add `case_sensitive: false` here for mysql.

Reply:
case_sensitive option used.